project-icp/bee-pollinator-app
# Gut Analysis View

Connects #14 

Currently branched off of #12, the relevant commits are from `c197da5` on

- Removes all the frontend analyze tests
- Removes all the calls on the frontend for specific analyze tasks, eg `analyze/pointsource`, and each task's corresponding tab
- Leaves the `land` tab, but just shows the AOI info instead of the histogram
- Removes some Django templates `analyze.html` and `model.html` that I couldn't find being used anywhere

### Testing
Pull this branch. Assuming your VMs are running...
`./scripts/bundle.sh`

 On `localhost:8000`...
- Verify that after drawing a shape, you see only one tab in the analyze panel ("Land")
- There should still be the usual info about the selected area in the analyze panel
- Verify there are no network errors in the console
